modal/sandbox.py

```diff
@@ -320,7 +320,7 @@ async def create(
         ] = None,  # Experimental controls over fine-grained scheduling (alpha).
         client: Optional[_Client] = None,
         environment_name: Optional[str] = None,  # *DEPRECATED* Optionally override the default environment
-        wait_for_scheduled: bool = False,  # Enables waiting for the sandbox to be assigned to worker before returning
+        wait_until_running: bool = False,  # Enable waiting for the sandbox to be assigned to a worker before returning
```

Comment:
```suggestion
        wait_until_running: bool = False,  # Wait for the sandbox to start running before returning
```

The "worker" concept is part of our internal ontology but not something we really expose to users. And it's redundant to say "Enable ..." when you have a boolean parameter